### ISSUE TITLE
Define metadata with template key

### ISSUE THREAD

**Issue by Wolfsrudel** [cloud66-oss/trackman#14]
Is it possible to use the following code?
```
metadata:
  repository_home_internal: "${HOME}/workspace/repository/internal"
  repository_home_internal_common: "{{ index .Metdata \"repository_home_internal\" }}/common"
  repository_home_internal_specific: "{{ index .Metdata \"repository_home_internal\" }}/specific"
```
It should be like defining a variable with a pre-defined variable content.

**Comment by khash** [comment]
If you mean using templates within the meta data section itself, then it should be possible but I haven't tried it myself. Are you seeing issues there?

**Comment by Wolfsrudel** [comment]
I can't get it to work. :D

**Comment by khash** [comment]
You're right. This was an issue which is fixed in 1.0.4 (released). 1.0.4 also has a `parse` command to show the workflow after it being parsed to help with debugging workflows.

**Comment by Wolfsrudel** [comment]
Nice! Thanks a lot!
